training/level-2-command-line-interfaces/dragon-warrior/tmarsha1/file_ops.py

```diff
@@ -1,21 +1,125 @@
-"""This module provides various functions for operating on files"""
+"""
+This module provides various functions for operating on files
+
+Functions:
+    copy_files: copy file(s) to a specific location.
+    move_files: move file(s) to a specific location.
+"""
 
 import subprocess
 
 
-def copy_files(files: list, destination: str):
+class FolderNotFoundError(Exception):
+    def __init__(self, value):
+        self.value = value
+    def __str__(self):
+        return repr(self.value)
+        
+
+class InsufficientRightsError(Exception):
+    def __init__(self, value):
+        self.value = value
+    def __str__(self):
+        return repr(self.value)
+
+
+def _process_files(
+        command: str, 
+        options: str, 
+        file: str, 
+        destination: str
+) -> str:
```

Comment:
@TroyLMarshall Could just be Github, but it looks like the function parameters in the definition are double indented.  Also, standard style is not to put the closing parenthesis on a separate line.